src/scss/objects/_form-controls.scss

```diff
@@ -91,7 +91,7 @@
          display: inline-block;
          width: $formControl-width;
          height: $formControl-height;
-         border: $formControl-border-width solid $grey--lighter;
+         border: $formControl-border-width solid $grey--light;
```

Comment:
Should we be using `$color-border--interactive`?

Reply:
Same here – I'm not 100% sure, I think this will have it's own unique token when we switch to PIE aliases.